python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -245,6 +257,21 @@ def validate(self):
             raise ValueError("Must set appropriate extinctionCoeffs config.  See "
                              "config/hsc/extinctionCoeffs.py in obs_subaru for an example.")
 
+        # If a wired origin was included in the config, check that it actually lies on the wired P1 line
```

Comment:
When I do this, I get an error.  Traceback is:
```
Traceback (most recent call last):
  File "/home/lauren/LSST/pipe_analysis/bin/colorAnalysis.py", line 2, in <module>
    from lsst.pipe.analysis.colorAnalysis import ColorAnalysisTask
  File "/home/lauren/LSST/pipe_analysis/python/lsst/pipe/analysis/colorAnalysis.py", line 206, in <module>
    class ColorAnalysisConfig(Config):
  File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/Linux64/pex_config/16.0-5-gd0f1235+4/python/lsst/pex/config/config.py", line 108, in __init__
    setattr(cls, k, copy.deepcopy(v))
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 180, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 280, in _reconstruct
    state = deepcopy(state, memo)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 240, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 240, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 180, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "/software/lsstsw/stack3_20171023/python/miniconda3-4.3.21/lib/python3.6/copy.py", line 274, in _reconstruct
    y = func(*args)
  File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/Linux64/pex_config/16.0-5-gd0f1235+4/python/lsst/pex/config/config.py", line 800, in unreduceConfig
    config.loadFromStream(stream)
  File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/Linux64/pex_config/16.0-5-gd0f1235+4/python/lsst/pex/config/config.py", line 563, in loadFromStream
    exec(stream, {}, local)
  File "<string>", line 2, in <module>
AttributeError: module 'lsst.pipe.analysis' has no attribute 'colorAnalysis
```
This seems to be related to the references to `lsst.pipe.analysis.colorAnalysis.ColorTransform` within the `ivezicTransformsHSC` object.  I don't understand this well enough to "fix" it (let alone know if it needs fixing, or is a case of Paul cleverness).  I'm inclined to leave as is for now...perhaps this will come up in the stack-ifying process.

Reply:
OK to leave it